blazor/dropdown-list/how-to/disable-the-dropdownlist-popup-items-based-on-selection.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Disable the DropDown List Popup items based on selection | Syncfusion
+description: Learn how to disable the DropDown List popup items based on the selection in the Syncfusion Blazor DropDown List component.
+platform: Blazor
+control: DropDown List
+documentation: ug
+---
+
+# How to disable the DropDown List popup items based on selection
+
+The following example demonstrates how to disable the DropDown List popup items based on the selection in the DropDown List component. This can be achieved by adding the e-disabled class to the selected list.
```

Comment:
The content is not correct.
1. Need to explain the purpose of this disable use case;
2. Explain how we achieved in our code